tests/serve/mock/faker/test_data_faker.py

```diff
@@ -1,8 +1,17 @@
-from meeshkan.serve.mock.faker import fake_it
+from faker import Faker
+from http_types import RequestBuilder
+
+from meeshkan.serve.mock.faker import MeeshkanDataFaker
+from meeshkan.serve.mock.faker.schema_faker import MeeshkanSchemaFaker
 from meeshkan.serve.mock.matcher import valid_schema
+from meeshkan.serve.mock.storage import Storage
 
 
 def test_faker_1():
+
+    request = RequestBuilder.from_dict(dict(method="post",  protocol="http", path="/", host="api.com"))
+    faker = MeeshkanSchemaFaker(Faker(), request, Storage())
```

Comment:
The names of the classes are a bit confusing to me. What's the difference between a `Faker` and `MeeshkanSchemaFaker`? Is the latter something like a `StatefulFaker`? I think the `Meeshkan...` is maybe a bit extra in our code?

Reply:
Faker is an imported class from the faker package. We use it to generate random strings. Previously our Faker wasn't a class so the problem didn't exist. So I named our fakers with Meeshkan prefix. But probably we can keep it as BaseFaker, StatelessFaker, StatefulFaker.